docs/SUMMARY.md

```diff
@@ -27,6 +27,7 @@
   * [Commands](using-northstar/commands.md)
   * [Launch arguments](using-northstar/launch-arguments.md)
   * [Advanced](using-northstar/advanced.md)
+  * [Progression System](using-northstar/progression.md)
```

Comment:
I'm gonna switch the order of the two. Advanced feels like the sorta thing that should show up last :P

```suggestion
  * [Progression System](using-northstar/progression.md)
  * [Advanced](using-northstar/advanced.md)
```

Reply:
Fair enough